compositor_render/src/renderer.rs

```diff
@@ -197,8 +209,10 @@ impl WgpuCtx {
 
         let yuv_bind_group_layout = YUVTextures::new_bind_group_layout(&device);
         let rgba_bind_group_layout = RGBATexture::new_bind_group_layout(&device);
+        let bgra_bind_group_layout = BGRATexture::new_bind_group_layout(&device);
```

Comment:
let's leave it for now, but those specific layouts could have been part of the chrome code